src/main/java/model/BlindVertical.java

```diff
@@ -0,0 +1,117 @@
+package model;
+
+public class BlindVertical extends Blind{
+
+    private int width;
```

Comment:
need to remove fields exist in superclass

Reply:
done